user-interface/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -19,6 +19,16 @@
       "name": "org.jobrunr.database.datasource.password",
       "type": "java.lang.String",
       "description": "The password jobRunr uses for the datasource."
+    },
+    {
+      "name": "qbic.access-token.salt",
+      "type": "java.lang.String",
+      "description": "The salt used to encrypt personal access tokens."
+    },
+    {
+      "name": "qbic.access-token.iteration-count",
+      "type": "java.lang.String",
+      "description": "The number of iterations used in personal access token encryption."
     }
```

Comment:
Nicely done! However should this information also be provided in the Readme markdown file?

Reply:
I think not :D the readme is out of date already and those descriptors are easily parseable.